target/linux/kirkwood/base-files/lib/upgrade/platform.sh

```diff
@@ -11,6 +11,17 @@ platform_do_upgrade() {
 	local board="$(board_name)"
 
 	case "$board" in
+	ctera,c200-v1)
+		local idx="$(find_mtd_index active_bank)"
+
+		if [ -n "$idx" ]; then
+			CI_KERNPART="$(strings /dev/mtd$idx | grep bank)"
```

Comment:
How about `busybox strings`? Then we don't need to touch RAMFS_COPY_BIN.

Reply:
Good point. Done.